core/src/main/java/org/mule/runtime/core/api/config/bootstrap/ArtifactType.java

```diff
@@ -6,17 +6,25 @@
  */
 package org.mule.runtime.core.api.config.bootstrap;
 
+import static org.mule.runtime.api.i18n.I18nMessageFactory.createStaticMessage;
+
 import org.mule.runtime.api.exception.MuleRuntimeException;
-import org.mule.runtime.core.api.config.i18n.CoreMessages;
+
+import java.util.Set;
 
 /**
  * List the possible types of deployable artifacts
  *
  * @since 3.7.0
  */
 public enum ArtifactType {
-  APP("app"), DOMAIN("domain"), PLUGIN("plugin"), POLICY("policy"), SERVICE("service"), SERVER_PLUGIN("serverPlugin"), ALL(
-      "app/domain");
+  APP("app"), DOMAIN("domain"), PLUGIN("plugin"), POLICY("policy"), SERVICE("service"), SERVER_PLUGIN("serverPlugin"),
```

Comment:
I dont think that the enum must include plugin, service and server plugin, as those artifact do not have a mule context/registry so there is no place where the bootstrap properties can be applied.

Reply:
These values are also used to load the artifact descriptor, not just to build the Mulecontext